xsd/siri_utility/siri_types-v2.0.xsd

```diff
@@ -153,4 +153,13 @@ A string containing a phrase in a natural language name that requires at least o
   </xsd:annotation>
   <xsd:restriction base="xsd:nonNegativeInteger"/>
  </xsd:simpleType>
+ <xsd:simpleType name="PercentageType">
```

Comment:
Just a note: My CR71 New Mode extension for FM also uses Percentage.... we need to align the types we use at some point

Reply:
@hennoit and @Aurige 
- ArrivalBoardingActivity annot. was updated
- annot. of both XOrientationRelativeToQuay were updated (more detailed examples)
- PercentageType was updated according to discussion on basecamp